components/brave_vpn/resources/panel/components/toggle/style.ts

```diff
@@ -2,46 +2,50 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at https://mozilla.org/MPL/2.0/.
+
 import styled from 'styled-components'
+import Icon from '@brave/leo/react/icon'
+import ProgressRing from '@brave/leo/react/progressRing'
+
+export const Content = styled.div`
+  display: flex;
+  padding: var(--leo-spacing-2xl) var(--leo-spacing-m);
+  align-items: center;
+  gap: var(--leo-spacing-m);
+  align-self: stretch;
+`
 
 // Status indicator
 export const StatusBox = styled.div`
   display: flex;
   align-items: center;
-  margin: 20px 0 24px 0;
+  gap: 8px;
+  flex: 1 0 0;
 `
 
-export const StatusText = styled.span`
+export const StatusLabel = styled.span<{ color: string }>`
   font-family: Poppins;
-  font-size: 14px;
-  font-weight: 400;
-  line-height: 20px;
-  color: ${(p) => p.theme.color.text01};
-  letter-spacing: 0.04em;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 26px;
+  color: ${(p) => p.color};
 `
 
-export const ActiveIndicator = styled.span`
-  width: 6px;
-  height: 6px;
-  margin-right: 8px;
-  border-radius: 50%;
-  background: ${(p) => p.theme.color.successBorder};
+export const ActiveIndicator = styled(Icon)`
+  --leo-icon-size: 20px;
+  color: var(--leo-color-systemfeedback-success-icon);
 `
 
 export const InActiveIndicator = styled(ActiveIndicator)`
-  background: ${(p) => p.theme.color.disabled};
+  color: var(--leo-color-icon-default);
 `
 
 export const FailedIndicator = styled(ActiveIndicator)`
-  background: ${(p) => p.theme.color.warn};
+  color: ${(p) => p.theme.color.warn};
```

Comment:
note: this isn't a Nala color - is `--leo-color-systemfeedback-warning-text` maybe what you wanted?